topics/timeseries.md

```diff
@@ -66,6 +84,10 @@ these columns depends on the `event_class` in question and is documented in deta
 
 ### Events with class `addToList`
 
+This event is emitted when a user has added a bookmark to a list. Only in classic
+
+<span class="deprecated">DEPRECATED</span> you will find `list.edit` to contain more useful information about what has been added to a list and when.
```

Comment:
I'm not totally sure it's fair to deprecate this until we cover off the route to add bookmarks via the My Bookmarks page with a new event...

Reply:
Ok. left this as undeprecated.